package.json

```diff
@@ -3,21 +3,32 @@
   "main": "js/theme.js",
   "version": "0.4.3",
   "private": true,
-  "devDependencies": {
-    "bower": "^1.8.4",
+  "scripts": {
+    "start": "yarn webpack-dev-server --open --config webpack.dev.js",
+    "build": "yarn webpack --config webpack.prod.js"
+  },
+  "dependencies": {
+    "bourbon": "~4.3",
+    "bourbon-neat": "~1.7",
     "browserify": "^13.0.0",
```

Comment:
Do we still need the browserify dependency here?

Reply:
I shall try without it.